app/slashcommands/command_invite.go

```diff
@@ -137,17 +137,23 @@ func (i *InviteProvider) parseMessage(a *app.App, c request.CTX, args *model.Com
 	return targetUsers, targetChannels, ""
 }
 
-func (i *InviteProvider) getUserProfile(a *app.App, username string) *model.User {
-	userProfile, nErr := a.Srv().Store().User().GetByUsername(username)
-	if nErr != nil {
+func (i *InviteProvider) getUsersFromMention(a *app.App, username string) []*model.User {
```

Comment:
Maybe the variable `username` could be renamed because it's not necessarily ever a username, it could be a group name.